load-tests/ballerina_hello/scripts/run.sh

```diff
@@ -19,4 +19,4 @@
 set -e
 source base-scenario.sh
 
-jmeter -n -t "$scriptsDir/"http-get-request.jmx -l "$resultsDir/"original.jtl -Jusers="$concurrent_users" -Jduration=1200 -Jhost=ballerina-hello.default.svc.cluster.local -Jport=9090 -Jprotocol=http -Jpath=hello $payload_flags
```

Comment:
We can only run a job for 6 hours. We must use a separate workflow and trigger it from VM  to handle long-running tests.